CRM/Contribute/DAO/Contribution.php

```diff
@@ -609,14 +617,18 @@ static function &fields() {
             'optionEditPath' => 'civicrm/admin/options/contribution_status',
           )
         ) ,
-        'address_id' => array(
+        'contribution_address_id' => array(
```

Comment:
yep - add unique name or it clobbers contact address_id (it kind of already was doing that ....)